src/main/java/com/microsoft/graph/httpcore/TelemetryHandler.java

```diff
@@ -0,0 +1,38 @@
+package com.microsoft.graph.httpcore;
+
+import java.io.IOException;
+
+import com.microsoft.graph.httpcore.middlewareoption.TelemetryOptions;
+
+import okhttp3.Interceptor;
+import okhttp3.Request;
+import okhttp3.Response;
+
+public class TelemetryHandler implements Interceptor{
+
+	public static final String SDK_VERSION = "SdkVersion";
```

Comment:
Do these strings need to be public? are they getting used elsewhere in the code, or making them private is fine?

Reply:
Currently, they are not being used elsewhere but there can be multiple headers added by other components in "SdkVersion" header to facilitate that I made it public.